extensions/dialog/common/client.mjs

```diff
@@ -15,42 +15,113 @@ class DialogModal extends Modal {
 
     this.msgDom = document.getElementById(DIALOG_MSG_DIV);
     this.subjectDom = document.getElementById(DIALOG_SUBJECT_DIV);
-    this.btnContDom = document.getElementById(DIALOG_BUTTON_CONTAINER_DIV);    
+    this.btnContDom = document.getElementById(DIALOG_BUTTON_CONTAINER_DIV);
   }
 
-  /**
-   * Call to set the dialog to display a single choice dialog.
-   */
-  async displayAsSingleChoice(subject, message, buttonText) {
+  async displayDialogInternal(setupDom) {
```

Comment:
If this method is not designed for external calling, maybe prepend an underscore: `async _displayDialogInternal(setupDom) {`